src/Lecture3.hs

```diff
@@ -89,7 +91,11 @@ weekday to the second.
 >>> daysTo Friday Wednesday
 5
 -}
-daysTo = error "TODO"
+daysTo :: Weekday -> Weekday -> Int
+daysTo firstDay secondDay
+    | firstDay < secondDay = (length . enumFrom $ firstDay) - (length . enumFrom $ secondDay)
+    | firstDay > secondDay = (length . enumFrom $ firstDay) + (7 - length (enumFrom secondDay))
```

Comment:
You can use `fromEnum` function to implement this one in a slightly more efficient way 🙂